src/php/tests/unit_tests/ChannelTest.php

```diff
@@ -81,8 +81,12 @@ public function testWatchConnectivityState()
     {
         $this->channel = new Grpc\Channel('localhost:0',
              ['credentials' => Grpc\ChannelCredentials::createInsecure()]);
-        $time = new Grpc\Timeval(1000);
-        $state = $this->channel->watchConnectivityState(1, $time);
+        // 'true' will attempt connecting the channel.
+        $lastState = $this->channel->getConnectivityState(true);
+
+        $now = Grpc\Timeval::now();
+        $deadline = $now->add(new Grpc\Timeval(5000000));
+        $state = $this->channel->watchConnectivityState($lastState, $deadline);
         $this->assertTrue($state);
         unset($time);
```

Comment:
Do I need to unset `$now` and the `new Grpc\Timeval(5000000)`? (not a php person).